pkg/apis/core/types.go

```diff
@@ -1434,11 +1434,11 @@ type KeyToPath struct {
 	Mode *int32
 }
 
-// Local represents directly-attached storage with node affinity
+// Local represents directly-attached storage with node affinity (Beta feature)
 type LocalVolumeSource struct {
-	// The full path to the volume on the node
-	// For alpha, this path must be a directory
-	// Once block as a source is supported, then this path can point to a block device
+	// The full path to the volume on the node, it can be a a disk, partition or directory.
```

Comment:
Perhaps "it can be either a directory or block device (disk, partition, ...)" - someone might use RAID or LVM. Kubernetes does not care what kind of block device it is.

Reply:
We should also mention block support is alpha.